### DISCUSSION TITLE
Create screenshots from media files

### DISCUSSION THREAD

**Discussion by sabersalv** [jesec/flood#265] [Ideas]
Most sites requires mediainfo and 3 screenshots to upload, and Flood already support mediainfo, and create torrent file. If Flood can support create screenshots, then you can use Flood alone to do the upload torrent job, it's super convenient.

**Comment by jesec** [comment]
It is on the roadmap: https://github.com/jesec/flood/projects/5#card-52268074 .

However, I am still undecided over implementation details. It could be too heavy to embed ffmpeg.
